matpower/matpower-converter/src/test/java/com/powsybl/matpower/converter/MatpowerImporterTest.java

```diff
@@ -136,4 +149,52 @@ private void testNetwork(Network network, String id) throws IOException {
     private void testNetwork(Network network) throws IOException {
         testNetwork(network, network.getId());
     }
+
+    private void testCaseSolved(MatpowerModel model) throws IOException {
+        String caseId = model.getCaseName();
+        Path matFile = tmpDir.resolve(caseId + ".mat");
+        MatpowerWriter.write(model, matFile);
+
+        Network network = new MatpowerImporter().importData(new FileDataSource(tmpDir, caseId), NetworkFactory.findDefault(), null);
+        testSolved(network);
+    }
+
+    private void testSolved(Network network) throws IOException {
+        // Precision required on bus balances (MVA)
+        double threshold = 0.0000001;
+        ValidationConfig config = loadFlowValidationConfig(threshold);
+        Path work = Files.createDirectories(fileSystem.getPath("/lf-validation" + network.getId()));
+        computeMissingFlows(network, config.getLoadFlowParameters());
+        assertTrue(ValidationType.BUSES.check(network, config, work));
+    }
+
+    private static ValidationConfig loadFlowValidationConfig(double threshold) {
+        ValidationConfig config = ValidationConfig.load();
+        config.setVerbose(true);
+        config.setThreshold(threshold);
+        config.setOkMissingValues(false);
+        LoadFlowParameters lf = new LoadFlowParameters();
+        lf.setTwtSplitShuntAdmittance(true);
+        config.setLoadFlowParameters(lf);
+        return config;
+    }
+
+    private static void computeMissingFlows(Network network, LoadFlowParameters lfparams) {
```

Comment:
Just to understand well: here we are performing a loadflow validation to check that the solved case is well filled. In the parameter of the loadflow validation, I understand that the slack is distributed (or not ?) on generator. So why do we fill missing value with `targetP` ? And maybe some generators have gone from PV to PQ, so why do we fill the Q of the terminal with targetQ. How does it work ?

Reply:
I modified the code to be more clear. We only set the `SSH` values in loads and generators  if the `P` and `Q` terminal values  are `NaN` (not initialized). We import a solved case and we perform a loadflow validation that fills the flows of the branches using the received voltages and angles and calculates a balance in all the buses.

Maybe we can include both setting loops (load and generators) inside the `LoadFlowResultsCompletion` method. We can do it in a new PR as we have to review all the `LoadFlowResultsCompletion` references.

Reply:
When you say "a solved case", what are the parameters of the load flow computation ? How was the slack distributed ? It was solved by open load flow ? It was the LoadFlow default parameter except the split shunt admittance parameter to true ?

Reply:
I solved on the `matpower` loadflow and I have not specified any parameter so the case was solved with the default matpower parameters.